### ISSUE TITLE
Header Missing

### ISSUE THREAD

**Issue by thebaaron42** [burkybang/SuiteAdvanced-Feedback#128]
The header section is missing on one of my PCs but not the other.  

Is there a way to reset the extension, or otherwise clear it to fix?  This bug has been going on for a couple of weeks, and I do believe it was in the last version as well.

![image](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/131487/73bdc589-3f5e-472d-9312-35d2e0235528)

**Comment by burkybang** [comment]
Try using the **Clear Cache** button in SuiteAdvanced settings.

![image](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/927886/67cf3474-352c-4a73-bcd1-691caffc132c)

**Comment by burkybang** [comment]
Okay, yes I've seen this issue before. You are only the third user to experience it over the span of months. I spent hours with one user trying to figure out the cause. After uninstalling and reinstalling SuiteAdvanced, the issue was resolved. I still don't know what the cause is, but reinstalling should work. Be sure to back up any of your settings first using the backup option in settings if needed.
